all.bib

```diff
@@ -6011,6 +6011,220 @@ @inproceedings{gyongyi2004combating
+  title = {Further advances in open domain dialog systems in the Fourth Alexa Prize SocialBot Grand Challenge},
+  year = {2021},
+}
+@article{thoppilan2022lambda,
+  author = {Romal Thoppilan and Daniel De Freitas and Jamie Hall and Noam Shazeer and Apoorv Kulshreshtha and Heng{-}Tze Cheng and Alicia Jin and Taylor Bos and Leslie Baker and Yu Du and YaGuang Li and Hongrae Lee and Huaixiu Steven Zheng and Amin Ghafouri and Marcelo Menegali and Yanping Huang and Maxim Krikun and Dmitry Lepikhin and James Qin and Dehao Chen and Yuanzhong Xu and Zhifeng Chen and Adam Roberts and Maarten Bosma and Yanqi Zhou and Chung{-}Ching Chang and Igor Krivokon and Will Rusch and Marc Pickett and Kathleen S. Meier{-}Hellstern and Meredith Ringel Morris and Tulsee Doshi and Renelito Delos Santos and Toju Duke and Johnny Soraker and Ben Zevenbergen and Vinodkumar Prabhakaran and Mark Diaz and Ben Hutchinson and Kristen Olson and Alejandra Molina and Erin Hoffman{-}John and Josh Lee and Lora Aroyo and Ravi Rajakumar and Alena Butryna and Matthew Lamm and Viktoriya Kuzmina and Joe Fenton and Aaron Cohen and Rachel Bernstein and Ray Kurzweil and Blaise Aguera{-}Arcas and Claire Cui and Marian Croak and Ed H. Chi and Quoc Le},
+  journal = {CoRR},
+  title = {LaMDA: Language Models for Dialog Applications},
+  volume = {0},
+  year = {2022},
+}
+@inproceedings{smith2022human,
+  author = {Eric Smith and Orion Hsu and Rebecca Qian and Stephen Roller and Y-Lan Boureau and Jason Weston},
+  booktitle = {Proceedings of the 4th Workshop on NLP for Conversational AI},
+  pages = {77--97},
+  title = {Human Evaluation of Conversations is an Open Problem: comparing the sensitivity of various methods for evaluating dialogue agents},
+  year = {2022},
+}
+@article{konstan2004introduction,
+  author = {Joseph A Konstan},
+  journal = {ACM Transactions on Information Systems (TOIS)},
+  number = {1},
+  pages = {1--4},
+  title = {Introduction to recommender systems: Algorithms and evaluation},
+  volume = {22},
+  year = {2004},
+}
+@article{hancock2020communication,
+  author = {Jeffrey T Hancock and Mor Naaman and Karen Levy},
+  journal = {Journal of Computer-Mediated Communication},
+  title = {{AI}-mediated communication: Definition, research agenda, and ethical considerations},
+  year = {2020},
+}
+@article{goldberg1992using,
+  author = {David Goldberg and David Nichols and Brian M Oki and Douglas Terry},
+  journal = {Communications of the ACM},
+  number = {12},
+  pages = {61--70},
+  title = {Using collaborative filtering to weave an information tapestry},
+  volume = {35},
+  year = {1992},
+}
+@article{white2009exploratory,
+  author = {Ryen W White and Resa A Roth},
+  journal = {Synthesis lectures on information concepts, retrieval, and services},
+  number = {1},
+  pages = {1--98},
+  title = {Exploratory search: Beyond the query-response paradigm},
+  volume = {1},
+  year = {2009},
+}
+@article{kelly2009methods,
```

Comment:
Why is this added, but removed from the rb file?

Reply:
In trying to fix my errors, I accidentally duplicated all of my entries. I went ahead and removed the duplicates - although I see they're not visible on the pull request.

Reply:
I'm not entirely sure either what's happening, because all the entries are showing up in my local copy of the .rb file and have compiled in the all.bib file, but are not showing up on the pull request.